src/rdkafka_admin.c

```diff
@@ -6280,14 +6280,14 @@ rd_kafka_admin_DescribeGroupsRequest(rd_kafka_broker_t *rkb,
                                      void *opaque) {
         int i;
         char *group;
-        size_t groups_cnt       = rd_list_cnt(groups);
+        int groups_cnt          = rd_list_cnt(groups);
         const char **groups_arr = calloc(groups_cnt, sizeof(*groups_arr));
         RD_LIST_FOREACH(group, groups, i) {
```

Comment:
Please put some air between logical sections of code;
```c
  variable declarations;

  some code();
  that does things();

  some other code that does other
  things();

  return;
```

Reply:
done